cassovary-core/src/main/scala/com/twitter/cassovary/graph/DirectedPathCollection.scala

```diff
@@ -13,10 +13,11 @@
  */
 package com.twitter.cassovary.graph
 
-import it.unimi.dsi.fastutil.objects._
+import com.twitter.cassovary.util.collections.{CQueue, Implicits, Order}
 import it.unimi.dsi.fastutil.ints._
-import java.util.Comparator
+import it.unimi.dsi.fastutil.objects._
 
+import Implicits._
```

Comment:
it's not that great to have to import Implicits._ everytime we have to use CQueue.... Any solution?